components/grid/style/mixin.less

```diff
@@ -3,47 +3,49 @@
 // mixins for grid system
 // ------------------------
 
+@prefix-cls: ~'@{ant-prefix}';
+
 .loop-grid-columns(@index, @class) when (@index > 0) {
-  .@{ant-prefix}-col@{class}-@{index} {
+  .@{prefix-cls}-col@{class}-@{index} {
```

Comment:
We should use `@col-prefix-cls` and `@row-prefix-cls` defined in https://github.com/ant-design/ant-design/pull/33969/

Reply:
Right that's better, I changed it. It compiles and seems to work. However I'm wondering: should imports get updated, or should the definition of the class should move to `mixin.less` ?

Reply:
Right, @row-prefix-cls and @col-prefix-cls should be defined in mixin.less and be imported in index.less.

Reply:
Done. I imported it in `rtl.less` as well. Overwrite of the mixins works fine.